client-cli/src/main/java/org/eclipse/dataspaceconnector/registration/cli/AddParticipantCommand.java

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.registration.cli;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.eclipse.dataspaceconnector.registration.client.models.Participant;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.ParentCommand;
+
+import java.util.concurrent.Callable;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@Command(name = "add")
+class AddParticipantCommand implements Callable<Integer> {
+
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+    private static final Logger LOGGER = Logger.getLogger(AddParticipantCommand.class.getName());
+
+    @ParentCommand
+    private ParticipantsCommand parent;
```

Comment:
```suggestion
    private ParticipantsCommand command;
```
it's a bit confusing why it's `parent.parent' in line 49
Can we rename this field to `command` and RegistrationServiceCli in ParticipantsCommand to `cli` and then we will have
`command.cli.registryApiClient.addParticipant(participant);` ?

The annotations indicate already the parent - child relationship so there's no need to repeat them in the field names I think

Reply:
Updated. Thanks